tests/pactHelper.js

```diff
@@ -10,11 +10,40 @@ const validAuthHeaders = {
   authorization: 'Basic ' + validUserPasswordHash,
   Origin: origin
 }
-const ocsSuccessMeta = ' <meta>\n' +
-  '  <status>ok</status>\n' +
-  '  <statuscode>100</statuscode>\n' +
-  '  <message/>\n' +
-  ' </meta>\n'
+
+const ocsMeta = function (status, statusCode, Message = null) {
+  if (Message == null) {
+    return ' <meta>\n' +
+      '  <status>' + status + '</status>\n' +
+      '  <statuscode>' + statusCode + '</statuscode>\n' +
+      '  <message/>\n' +
+      ' </meta>\n'
+  }
+  return ' <meta>\n' +
+    '  <status>' + status + '</status>\n' +
+    '  <statuscode>' + statusCode + '</statuscode>\n' +
+    '  <message>' + Message + '</message>\n' +
+    ' </meta>\n'
+}
+
+const shareResponseOcsData = function (shareType, id, permissions, fileTarget) {
+  const data = '  <id>' + id + '</id>\n' +
+    '  <share_type>' + shareType + '</share_type>\n' +
+    '  <uid_owner>admin</uid_owner>\n' +
+    '  <displayname_owner>admin</displayname_owner>\n' +
+    '  <permissions>' + permissions + '</permissions>\n' +
+    '  <uid_file_owner>admin</uid_file_owner>\n' +
+    '  <displayname_file_owner>admin</displayname_file_owner>\n' +
+    '  <path>' + fileTarget + '</path>\n' +
+    '  <file_target>' + fileTarget + '</file_target>\n'
+
+  if (shareType === 3) {
+    return data +
+      '  <url>' + config.owncloudURL + '/yrkoLeS33y1aTya</url>\n'
```

Comment:
will that token always be the same?

Reply:
No, it is usually different for different response. But that does not really matter, IMO. All that is checked is if the url is a string.